### PR TITLE
redirects user to 'next' url

### PR DESCRIPTION
**Pull request by rodolfotorres02**
Will try to get the next url parameter and redirect user to that link, otherwise, keep normal behavior

### CODE REVIEW

File: loginas/views.py [hunk 1]
```diff
@@ -82,4 +83,9 @@ def user_logout(request):
     """
     restore_original_login(request)
 
-    return redirect(la_settings.LOGOUT_REDIRECT)
+    try:
+        url = request.META.get("HTTP_REFERER", "/").split('?next=')[1]
+    except IndexError:
+        url = settings.LOGINAS_LOGOUT_REDIRECT_URL
```
Comment (skorokithakis) [inline loginas/views.py:89]:
```suggestion
        url = la_settings.LOGOUT_REDIRECT
```

File: loginas/views.py [hunk 2]
```diff
@@ -82,4 +83,9 @@ def user_logout(request):
     """
     restore_original_login(request)
 
-    return redirect(la_settings.LOGOUT_REDIRECT)
+    try:
+        url = request.META.get("HTTP_REFERER", "/").split('?next=')[1]
```
Comment (skorokithakis) [inline loginas/views.py:87]: There's an issue here, when the Referer has not been specified, this will go to `/` instead of `LOGOUT_REDIRECT`.

Comment (skorokithakis) [reply loginas/views.py:87]: Oh, no, I read it wrong, sorry.

**Review by skorokithakis — APPROVED** [review]
Thank you!

File: loginas/views.py [hunk 3]
```diff
@@ -1,3 +1,4 @@
+from django.conf import settings
```
Comment (skorokithakis) [inline loginas/views.py:1]:
```suggestion
```

**Comment by rodolfotorres02** [comment]
Happy to help 💪🏻

**Pull request merged**